test/scripts/hexo/post.ts

```diff
@@ -1534,7 +1534,7 @@ describe('Post', () => {
         engine: 'markdown'
       });
       should.fail();
-    } catch (err) {}
+    } catch (_) {}
```

Comment:
The block can be removed 

```suggestion
    } catch {}
```